docs/hardware.md

```diff
@@ -68,6 +64,7 @@ Quick intro:
                             <li>Raspberry Pi 4 Model B<br>Compute Module 4</li>
                             <li>Raspberry Pi 3 Model A+ / B / B+<br>Compute Module 3 / 3+</li>
                             <li>Raspberry Pi 2 Model B v1.1 / B v1.2</li>
+                            <li>Raspberry Pi Zero 2 W</li>
```

Comment:
Doubled, it's already at the top of the list 🙂.

Reply:
True. Removed, just wanted to be sure... :)